### PR TITLE
Set the default select value for filterOptions

### PR DESCRIPTION
**Pull request by Rajat-Dabade**
#### Summary
This PR will set the default value for the filter option in boards. 
Example: For the `text` field the default will be `is` and for the `options` field it will be `includes`

#### Ticket Link
Fixes https://github.com/mattermost/focalboard/issues/4206

### CODE REVIEW

File: webapp/src/components/viewHeader/__snapshots__/filterComponent.test.tsx.snap [hunk 1]
```diff
@@ -1013,7 +1013,7 @@ exports[`components/viewHeader/filterComponent return filterComponent and filter
             type="button"
           >
             <span>
-              includes
+              (unknown)
```
Comment (Pinjasaur) [inline webapp/src/components/viewHeader/__snapshots__/filterComponent.test.tsx.snap:1016]: @Rajat-Dabade is this an expected side effect of the change?

Comment (Rajat-Dabade) [reply webapp/src/components/viewHeader/__snapshots__/filterComponent.test.tsx.snap:1016]: @Pinjasaur, Yes this is expected.

**Review by Pinjasaur — APPROVED** [review]
LGTM 👍🏻

**Review by harshilsharma63 — APPROVED** [review]
LGTM 👍 Nice UX improvement!

**Pull request merged**